src/sentry/issues/endpoints/organization_group_search_views.py

```diff
@@ -0,0 +1,68 @@
+from rest_framework import status
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.organization import OrganizationEndpoint, OrganizationPermission
+from sentry.api.paginator import SequencePaginator
+from sentry.api.serializers import serialize
+from sentry.api.serializers.models.groupsearchview import GroupSearchViewSerializer
+from sentry.models.groupsearchview import GroupSearchView
+from sentry.models.organization import Organization
+
+DEFAULT_VIEWS = [
```

Comment:
Can we define this in such a way (with types, or by using the serializer) so that we can guarantee that the keys here are correct? I'm worried that we might add a column and forget to modify this